src/actors/realtime_update_actors.rs

```diff
@@ -124,11 +157,13 @@ impl RealTimeReloader {
 
     fn apply_rt(&self, dataset: Arc<Dataset>) -> Result<(), Error> {
         //TODO: make this async
-        let gtfs_rt = fetch_gtfs_rt(&self.gtfs_rt_url)?;
-
-        // we compute the new timetable
-        let rt_dataset = self.make_rt_dataset(dataset, gtfs_rt)?;
+        let gtfs_rts = self
+            .gtfs_rt_urls
+            .iter()
+            .filter_map(|url| fetch_gtfs_rt(&url)) // All failing will be logged in fetch_gtfs
+            .collect();
```

Comment:
for all those mechanism, shouldn't we have error if we fail to load all the gtfs_rts ?

Reply:
I don't know, here we can serve what we are able to read.

Reply:
yes I do aggree that it's nice if 1 feed out of 2 fails, but my point is that is all the feeds fails, we won't have a hard error